recipe/meta.yaml

```diff
@@ -26,31 +32,34 @@ build:
 requirements:
   build:
     - python  # [win]
+    - 7za  # [win]
+    - curl  # [win]
+    - msinttypes  # [win and py27]
     - perl 5.22.2.1
-    - freetype 2.7|2.7.*
-    - fontconfig 2.12.*
+    - freetype 2.7|2.7.*  # [unix]
+    - fontconfig 2.12.*  # [linux]
     - openssl 1.0.*
     - jpeg 9*
     - zlib 1.2.*
     - libpng >=1.6.28,<1.7
-    - gst-plugins-base
+    - gst-plugins-base  # [linux]
     - icu 58.*
-    - libxcb
-    - dbus
+    - libxcb  # [linux]
```

Comment:
Before @pkgw started packaging X11 on all platforms I would have agreed we can just ignore X11 anywhere but Linux. However, it has since become clear to me that people are very interested in a working X11 ecosystem on all platforms. So I think it is fair for them to have the opportunity to weigh in. Personally I have no stake in this.